flake.nix

```diff
@@ -0,0 +1,23 @@
+{
+  description = "Nix Flake for kf-website";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      with import nixpkgs { inherit system; };
+      let project = pkgs.callPackage ./yarn-project.nix { } { src = ./.; };
+      in {
+
+        defaultPackage = project.overrideAttrs (oldAttrs: {
+          buildInputs = oldAttrs.buildInputs ++ [ pkgs.python3 ];
```

Comment:
Do you need actually need python here?

Reply:
Yes, python3 is needed to build node-gyp modules (`lmdb-store` and `msgpackr-extract`)